adagucserverEC/CSLDPostRequest.cpp

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
Maybe it is better to remove CSLDPostRequest files, as they are not yet implemented